### ISSUE TITLE
PayMillSDK.Framework has symbolic link to Resources which does not exist

### ISSUE THREAD

**Issue by chrisdoc** [paymill/paymill-ios#12]
Running ```pod install``` consecutive will result into the following error message

```
Errno::ENOENT - No such file or directory @ rb_file_s_stat - ../Pods/PayMillSDK/samples/vouchermill/PayMillSDK/PayMillSDK.framework/Resources
/usr/local/Cellar/ruby/2.2.3/lib/ruby/2.2.0/fileutils.rb:904:in `stat'
/usr/local/Cellar/ruby/2.2.3/lib/ruby/2.2.0/fileutils.rb:904:in `symbolic_modes_to_i'
/usr/local/Cellar/ruby/2.2.3/lib/ruby/2.2.0/fileutils.rb:951:in `fu_mode'
/usr/local/Cellar/ruby/2.2.3/lib/ruby/2.2.0/fileutils.rb:1025:in `block (2 levels) in chmod_R'
/usr/local/Cellar/ruby/2.2.3/lib/ruby/2.2.0/fileutils.rb:1477:in `preorder_traverse'
/usr/local/Cellar/ruby/2.2.3/lib/ruby/2.2.0/fileutils.rb:1023:in `block in chmod_R'
/usr/local/Cellar/ruby/2.2.3/lib/ruby/2.2.0/fileutils.rb:1022:in `each'
/usr/local/Cellar/ruby/2.2.3/lib/ruby/2.2.0/fileutils.rb:1022:in `chmod_R'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer.rb:117:in `block in prepare'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/user_interface.rb:140:in `message'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer.rb:116:in `prepare'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer.rb:103:in `install!'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/command/project.rb:156:in `run'
/usr/local/lib/ruby/gems/2.2.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/command.rb:48:in `run'
/usr/local/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/bin/pod:44:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'
```
If we look into PayMillSDK.framework we can see that there is a symbolic link from Resources to Versions/Current/Resources
```bash
PayMillSDK.framework git:(master) ls -la
total 24
drwxr-xr-x  6 ch  staff  204 May 18 15:54 .
drwxr-xr-x  3 ch  staff  102 May 18 15:54 ..
lrwxr-xr-x  1 ch  staff   24 May 18 15:54 Headers -> Versions/Current/Headers
lrwxr-xr-x  1 ch  staff   27 May 18 15:54 PayMillSDK -> Versions/Current/PayMillSDK
lrwxr-xr-x  1 ch  staff   26 May 18 15:54 Resources -> Versions/Current/Resources
drwxr-xr-x  4 ch  staff  136 May 18 15:54 Versions
```

Further down the rabbit hole we can see that Versions/Current/Resources can't be found because Version/Current only contains Headers and the PayMillSDK

```bash
 PayMillSDK.framework git:(master) ls -la Versions/Current/
total 8464
drwxr-xr-x   4 ch  staff      136 May 18 15:54 .
drwxr-xr-x   4 ch  staff      136 May 18 15:54 ..
drwxr-xr-x  13 ch  staff      442 May 18 15:54 Headers
-rw-r--r--   1 ch  staff  4330856 May 18 15:54 PayMillSDK
```

**Comment by yalnazov** [comment]
Hi @chrisdoc, 

thanks for the report, I will check this asap and get back to you.

**Comment by yalnazov** [comment]
Hi @chrisdoc , 

symbolic link to Resources is removed and after I did a local reset on my cocoapods 0.38.2 consecutive **pod install** does not lead to the reported error. I tested with cocoapods 0.39.0 as well and it works too. Having a symbolic link to a non-existing folder was for sure wrong, so this was a nice catch. Interestingly cocoapods 0.38.2 has introduced the issue of not being able to replace a Pod with a broken symbolic link in it. I think this is now fixed in 0.39.0. Here is more about it - https://github.com/CocoaPods/CocoaPods/issues/3914
